src/openfermion/measurements/_prony.py

```diff
@@ -0,0 +1,54 @@
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+"""An implementation of Prony's method (or the matrix pencil method)
+This fits a signal f(t) to sum_i=1^M a_i gamma_i^t, where a_i, gamma_i
+are complex numbers
+"""
+import scipy
+import numpy
+
+
+def prony(signal):
+    """Estimates amplitudes and phases of a sparse signal using Prony's method
```

Comment:
Nit: There should be a one-line docstring, then a paragraph. I.e.

```
"""One-line docstring here.

Paragraph here.

Args:
...
```

Reply:
Ah, thanks for spotting that.